transports/p2p/addrmgr/addrmanager.go

```diff
@@ -134,6 +135,10 @@ const (
 	// getAddrPercent is the percentage of total addresses known that we
 	// will share with a call to AddressCache.
 	getAddrPercent = 23
+
+	// defaultBantime is the default amount of time for which to ban
+	// a peer that we cannot connect to.
+	defaultBanTime = time.Hour * 24
```

Comment:
I'm wondering if this `defaultBanTime` value can be changed by user or configuration? If not, maybe a name: `banTime` would be better.

Reply:
Changed to `banTime` :)